resources/geocoding/en/52.txt

```diff
@@ -0,0 +1,418 @@
+# Copyright (C) 2017 Pindrop Security, Inc.
+
+# Licensed under the Google Software Grant and Corporate Contributor License Agreement (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# https://cla.developers.google.com/clas/new?domain=DOMAIN_GOOGLE&kind=KIND_CORPORATE
+
+# Generated from:
+# https://www.itu.int/dms_pub/itu-t/oth/02/02/T020200008A0002PDFS.pdf
+# Internal statistics data (2015-07-10).
+# In cases of multiple areas for same number:
+# 	If all areas in same state, default to state
+# 	If areas in different states, default to largest population and
+# 	  name both states
+
+52223|Puebla
+52224|Puebla
+52225|Tlapacoyan, VER
+52226|Altotonga/Jalacingo, VER
+52227|Huejotzingo/San Buenaventura Nealtican, PUE
+52228|Jalapa/Tuzamapan, VER
+52229|Veracruz, VER
+52231|Teteles/Teziutlán, PUE
+52232|La Vigueta/Martínez de la Torre, VER
+52233|Puebla
+52235|Veracruz
+52236|Teotitlán del Camino, OAX/PUE
```

Comment:
This tells me that it is the city Teotitlán del Camino, which is in two states, OAX and PUE: I assume this is not intended? If it is in two states, just name both of them, in full.